CONTRIBUTING.md

```diff
@@ -108,7 +108,7 @@ corresponding `1.x` version of `spyder-kernels`.
 This can be done via two methods: installing the correct version via `conda`:
 
 ```bash
- conda install spyder-kernels=1
+conda install -c conda-forge spyder-kernels=1.*
```

Comment:
As I mentioned in the spyder-docs PR, please change this to

    conda install -c spyder-ide spyder-kernels=1.*

Reply:
Yup